subscription-engine/kafka-ws/app/Kafka/KafkaConsumer.js

```diff
@@ -24,10 +25,15 @@ class KafkaConsumer {
     this.consumer = new Consumer(consumerConfig);
     // only one callback by topic
     this.registeredCallbacks = new Map();
+
+    // healthChecker
+    this.healthCheckerBind = this.healthChecker.bind(this);
+    StateManager.registerService('kafka');
```

Comment:
it makes more sense to keep this:
`StateManager.registerService('kafka');`

inside StateManager.js, because that will never change.